libraries/AP_HAL_PX4/CAN.h

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>
+ *
+ * With modifications for Ardupilot CAN driver
+ * Copyright (C) 2017 Eugene Shamaev
+ */
+
+#pragma once
+
+#include "AP_HAL_PX4.h"
+#include <systemlib/perf_counter.h>
+#include <AP_HAL/CAN.h>
+#include <pthread.h>
+#include <semaphore.h>
+
+#include "bxcan.h"
+#include "AP_HAL/utility/RingBuffer.h"
+
+#define UAVCAN_STM32_NUM_IFACES 2
```

Comment:
It is a legacy from original stm32 can driver taken from uavcanlib.
Fixed in all places